//package com.sincetimes.redwar.game.comnunicate.protocol;
message RceObtainAskForHelp
{
	optional int32 ret = 1;
}